OpenTrons/opentrons-api
# refactor(protocol_designer): move Slideout from atoms to components

# Overview
move Slideout from atoms to components and this is prep work for PD

close AUTH-627


## Test Plan and Hands on Testing
- go to sandbox
- create a protocol for OT-2/Flex
- go to design and open off deck labware slideout
check can edit off deck labware



## Changelog



## Review requests



## Risk assessment
low
